proposals/cloudedgeclientCG/charter.md

```diff
@@ -38,7 +38,7 @@ The Group will have succeeded if it can achieve the following:
 No Specifications will be produced under the current charter.
 
 ### Non-Normative Reports
-The group may produce other Community Group Reports within the scope of this charter that are not Specifications, for instance use cases, requirements, or white papers.
+The group may produce other Community Group Reports within the scope of this charter that are not Specifications, for instance use cases, requirements, white papers, or draft API or protocol proposals.
```

Comment:
see https://github.com/w3c/web-networks/pull/24#discussion_r1657009829 re discussion on exclusing specs from scope.

In general, I would find it hard to have an API or a protocol that shouldn't be considered a specification (i.e. a document that describes how two implementations can interoperate).

Reply:
done, reverted

Reply:
We can always add this in a different PR, but I'd like to at least get this one merged.